### PR TITLE
Feature - Disabled options

### PR DESCRIPTION
**Pull request by joefitter**
* Allow checkbox and radios to be disabled

### CODE REVIEW

File: src/forms/radio-group.jsx [hunk 1]
```diff
@@ -57,6 +57,7 @@ class RadioGroup extends MultipleChoice(Input) {
                   type="radio"
                   name={this.props.name}
                   value={opt.value}
+                  disabled={opt.diabled}
```
Comment (lennym) [inline src/forms/radio-group.jsx]:
```suggestion
                  disabled={opt.disabled}
```

File: src/types/options.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ export default PropTypes.arrayOf(
   PropTypes.oneOfType([
     PropTypes.shape({
       value,
-      label: PropTypes.oneOfType([PropTypes.string, PropTypes.node])
+      label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+      disabled: PropTypes.boolean
```
Comment (lennym) [inline src/types/options.js:9]: Will this beef if undefined?

Comment (joefitter) [reply src/types/options.js:9]: I assumed so

Comment (joefitter) [reply src/types/options.js:9]: oh sorry, you mean if disabled is undefined, no, no beef, it has a .isRequired prop to make it beef if undefined

**Pull request merged**